src/scss/index.scss

```diff
@@ -85,9 +85,11 @@ $color-bg--accept             : $green--offWhite;
 $color-bg--notification       : $yellow--light;
 $color-bg--info               : $blue--offWhite;
 $color-bg--error              : $red--offWhite;
+$color-bg--default            : $blue;
```

Comment:
What's the intended difference between this and `$color-bg`? (as I'd expect that to be the `default`)

Reply:
Well i need it to be blue but we don't have one. Open to naming suggestions... @ashleynolan

Reply:
Have you got a design for where this is used? (as may be able to link it to the component type)

Reply:
Not the design but it is for this class...

`.c-basketTray`

@ashleynolan